app/assets/javascripts/src/Metamaps.TopicCard.js

```diff
@@ -143,8 +147,11 @@ Metamaps.TopicCard = {
       loader.setRange(0.9); // default is 1.3
       loader.show() // Hidden by default
       var e = embedly('card', document.getElementById('embedlyLink'))
-      if (!e) {
+      if (!e && Metamaps.Erb.RAILS_ENV != 'development') {
```

Comment:
because when you are developing on a localhost server, the api call to embedly will always fail if the link you put in is a link to a map on your localhost. and when it fails, it automatically removes the link, which I didn't want